extras/retrofit2/pom.xml

```diff
@@ -0,0 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>async-http-client-extras-parent</artifactId>
+        <groupId>org.asynchttpclient</groupId>
+        <version>2.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>async-http-client-extras-retrofit2</artifactId>
+    <name>Asynchronous Http Client Retrofit2 Extras</name>
+    <description>The Async Http Client Retrofit2 Extras.</description>
+
+    <properties>
+      <retrofit2.version>2.3.0</retrofit2.version>
+      <lombok.version>1.16.16</lombok.version>
```

Comment:
I can convert all the code to normal java code, but frankly i don't think it's worth it - some other open source projects, namely spring-cloud started using lombok 2-3 years ago and it seem to work for them very well.

So, do you want me to de-lombok the code?